specification/search/data-plane/Azure.Search/preview/2021-04-30-Preview/searchindex.json

```diff
@@ -1180,6 +1229,125 @@
+          "type": "string",
+          "readOnly": "true",
+          "description": "The concatenated keyword fields."
+        }
+      },
+      "description": "The concatenated semantic fields sent to the reranker model.",
+      "x-ms-client-name": "RerankerInput"
+    },
+    "SemanticFieldResult": {
+      "x-ms-client-name": "Semantic",
+      "properties": {
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The name of this semantic field."
+        },
+        "state": {
+          "type": "string",
+          "readOnly": true,
+          "description": "How much of this semantic field was used during semantic processing.",
+          "enum": [
+            "used",
+            "unused",
+            "partial"
+          ],
+          "x-ms-enum": {
+            "name": "SemanticFieldResultState",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "used",
+                "name": "Used",
+                "description": "The field was fully used."
+              },
+              {
+                "value": "unused",
+                "name": "Unused",
+                "description": "The field was not used."
+              },
+              {
+                "value": "partial",
+                "name": "Partial",
+                "description": "The field was partially used."
+              }
+            ]
+          }
+        }
+      }
+    },
+    "DocumentSemanticDebugInfo": {
+      "x-ms-client-name": "Semantic",
+      "description": "Semantic debug info for this document.",
+      "properties": {
+        "titleField": {
+          "$ref": "#/definitions/SemanticFieldResult",
+          "description": "The title field used in semantic processing.",
+          "readOnly": true,
+          "x-ms-client-name": "TitleField"
+        },
+        "contentFields": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SemanticFieldResult"
+          },
+          "description": "The list of content fields used in semantic processing.",
+          "readOnly": true,
+          "x-ms-client-name": "ContentFields"
+        },
+        "keywordFields": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SemanticFieldResult"
+          },
+          "description": "The list of keyword fields used in semantic processing.",
+          "readOnly": true,
+          "x-ms-client-name": "KeywordFields"
+        },
+        "rerankerInput": {
+          "$ref": "#/definitions/RerankerInputResult",
+          "description": "The concatenated semantic fields sent to the reranker model.",
+          "readOnly": true,
+          "x-ms-client-name": "RerankerInput"
+        }
+      }
+    },
+    "DocumentDebugInfo": {
+      "x-ms-client-name": "DocumentDebugInfo",
+      "description": "Debug info for this document.",
+      "$ref": "#/definitions/DocumentSemanticDebugInfo"
```

Comment:
DocumentDebugInfo has a semantic property?

Reply:
Yep. I think the idea here was to make room for non-semantic debug properties if they pop up in the future.

Reply:
Doesn't this need a "properties" with "semantic" that refs to "DocumentSemanticDebugInfo"?